plugins/hydra_ray_launcher/tests/test_ray_aws_launcher.py

```diff
@@ -164,27 +164,17 @@ def upload_and_install_wheels(
 def validate_lib_version(yaml: str) -> None:
     # a few lib versions that we care about
     libs = ["ray", "cloudpickle", "pickle5"]
-    local_versions = set()
     for lib in libs:
-        v = pkg_resources.get_distribution(lib).version
-        local_versions.add(f"{lib}=={v}")
-
-    log.info(f"Locally running {local_versions}")
-    out, _ = _run_command(
-        [
-            "ray",
-            "exec",
-            yaml,
-            "pip freeze",
-        ]
-    )
-    remote_versions = out.split()
-    log.info(f"Remotely running {remote_versions}")
-    for local in local_versions:
-        if local not in remote_versions:
-            raise ValueError(
-                f"lib version not matching, local_version: {local} \nremote_versions: {remote_versions}"
-            )
+        local_version = f"{pkg_resources.get_distribution(lib).version}"
+        out, _ = _run_command(
+            [
+                "ray",
+                "exec",
+                yaml,
+                f"pip freeze | grep {lib}",
+            ]
+        )
+        assert local_version in out, f"{lib} version mismatch"
```

Comment:
moving the listing python version and installing conda to the main script, this way we get more up to date logging when running the script so it's easier to spot if anything goes wrong.

Reply:
commented on the wrong file, resolving.